docs/misc/java-install.md

```diff
@@ -129,11 +129,17 @@ prompt you to update like this:
 ![Pterodactyl Automatic Prompt](pterodactyl-prompt.png)
 
 If this does not show up for you, the Java version can be manually changed. Navigate to the
-"Startup" tab of your server, select `ghcr.io/pterodactyl/yolks:java_17` from the "Docker Image"
+"Startup" tab of your server, select `Java 21` from the "Docker Image"
 dropdown as shown in the image below.
 
 ![Pterodactyl Manual Java Version Change](pterodactyl-manual.png)
 
+:::note
+
+If you don't see `Java 21` in the dropdown, an administrator account is required to update the Paper egg.
```

Comment:
This should probably explain how, or link to a page on how to update it?

Reply:
> This should probably explain how, or link to a page on how to update it?

seems a little out of scope for Paper docs and most people using Pterodactyl are just customers of hosting companies, so that wouldn't be of use to them

could clarify that an administrator _of the panel_, i.e. their hosting company, needs to do it

Reply:
I pretty much copy pasted what the other note above had. So if there's something that needs changing I think it should be done to both of them.

The section is currently written towards Pterodactyl _users_ only. I could rewrite the section to include instructions for administrators too in another PR if desired.

Reply:
Not required